python/nano/test/tf/keras/test_trace_and_quantize.py

```diff
@@ -117,9 +115,7 @@ def test_attribute_access_after_quantize(self):
                                                       accelerator="openvino",
                                                       input_spec=tf.TensorSpec(shape=(None, 4), dtype=tf.float32),
                                                       x=np.random.random((100, 4)),
-                                                      y=np.random.random((100, 5)),
-                                                      accuracy_criterion = {'relative': 0,
-                                                                            'higher_is_better': True})
+                                                      y=np.random.random((100, 5)))
```

Comment:
why removing `accuracy_criterion` here?

Reply:
INC 2.0 will check "tolerable_loss" > -1 and < 1, if 'relative' = 0, then `tolerable_loss`=1

Reply:
It's necessary to test `accuracy_cirterion` here, we have other UT to test `accuracy_cirterion` argument, so I simply remove it

Reply:
Make sense, It's OK to remove it here if there are other uts cover `accuracy_criterion` parameter.